synapse/_scripts/synapse_port_db.py

```diff
@@ -236,9 +243,12 @@ def get_instance_name(self):
         return "master"
 
 
-class Porter(object):
-    def __init__(self, **kwargs):
-        self.__dict__.update(kwargs)
+class Porter:
```

Comment:
Ah looks like these never got some of the Python 3 upgrades, we should probably run `pyupgrade` on them...